<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationSelectionDialog</class>
 <widget class="QWidget" name="LocationSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Location Selection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="CtlLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CtlComboBox" name="comboBox">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CtlButton" name="btnOk">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="CtlButton" name="btnCancel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>horizontalSpacer</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>CtlButton</class>
      <extends></extends>
      <header>bus/common_control/src/button/CtlButton.h</header>
    </customwidget>
    <customwidget>
      <class>CtlComboBox</class>
      <extends></extends>
      <header>bus/common_control/src/input/CtlComboBox.h</header>
    </customwidget>
    <customwidget>
      <class>CtlLabel</class>
      <extends></extends>
      <header>bus/common_control/src/label/CtlLabel.h</header>
    </customwidget>
  </customwidgets>
 <resources/>
 <connections/>
</ui>
